index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
use https://validator.w3.org/ and check errors
![image](https://user-images.githubusercontent.com/35924318/71360848-cf844e80-2599-11ea-9e52-f265cf2f6865.png)